README.md

```diff
@@ -196,6 +196,12 @@ npx eslint .
 - 👤 Twitter: [@MAnwarHussaini](https://twitter.com/MAnwarHussaini)
 - 👤 LinkedIn: [Mohammad Anwar Hussaini](https://www.linkedin.com/in/mohammad-anwar-hussaini-876638267/)
 
+👤 **Andrea Manuel**
+
+- GitHub: [@AndreaM2429](https://github.com/AndreaM2429)
+- Twitter: [@twitter](https://twitter.com/AndreaManuelOr1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/andrea-manuel-2b075026a/)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
```

Comment:
- [ ] The MIT license link is broken, Please add the license file and check the link.

![image](https://github.com/M-Anwar-Hussaini/Awesome-Library/assets/68796232/089769af-762c-453a-9348-cf101b5c26c5)